lib/containers/docker.pm

```diff
@@ -32,4 +33,56 @@ sub configure_insecure_registries {
     record_info "setup $self->runtime", "deamon.json ready";
 }
 
+sub check_containers_firewall {
+    record_info "firewall", "Checking that firewall is enabled and properly configured";
+    my $container_name = 'sut_container';
+    my $docker_version = get_docker_version();
+    systemctl('is-active firewalld');
```

Comment:
IMO it's not obvious that this asserts that `firewalld` is running.

Could be made more obvious by adding a comment or specifying that explicitly:

```suggestion
    systemctl('is-active firewalld', ignore_failure => 0);
```